daffodil-test/src/test/resources/org/apache/daffodil/extensions/type_calc/typeCalcFunctions.tdml

```diff
@@ -281,5 +313,81 @@
     </tdml:infoset>
   </tdml:parserTestCase>
 
+  <tdml:parserTestCase name="sparse_enum_01"
+    root="sparse_enum01" model="inputTypeCalc-Embedded.dfdl.xsd" description="Extensions - inputTypeCalc keysetValue transform">
+
+    <tdml:document>
+    <tdml:documentPart type="byte">
+    00
```

Comment:
If you want this line per byte preserved, which I see you do given that each byte corresponds to an infoset item and those are vertically spread, you need to put <![CDATA[ .... ]]> around them. Then XML formatting won't screw with your whitespace.